api/actions/updateWorkItemPriority.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
             sails.hooks['issues-hook'].queue.push({
                 name: 'UPDATE_WORK_ITEM_PRIORITY',
                 priority,
-                relatedUrl: url
+                relatedUrl: html_url
```

Comment:
Thanks for this fix. Only suggestion I have is that we are moving from using Heroku Connect to using GUS API for the updates. Is that something you can do here? You can look at createWorkItem.js and updateIssue.js for how we do it. Slack me if you need help with that.

Reply:
Sure thing, I'm happy to look into it in a separate issue 👍 .